news/rustconf-2024-what-to-expect.md

```diff
@@ -0,0 +1,85 @@
+---
+title: 'RustConf 2024: What to Expect'
```

Comment:
Can we change the title to "What's in Store at RustConf 2024"? It's ok if the link reflects the original title.